testutils/src/main/java/com/microsoft/identity/internal/testutils/mocks/MockServerResponse.java

```diff
@@ -62,6 +62,56 @@ public static HttpResponse getMockTokenFailureInvalidGrantResponse() {
         return response;
     }
 
+    public static HttpResponse getMockCloudDiscoveryResponse() {
```

Comment:
nit: Not sure if instead of getting static data, we want to be able to get custom objects based on the some of the input parameters to this method. 
Such that we can get different mock data by just calling the method with different values.